plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/BasePinotConnectorSmokeTest.java

```diff
@@ -1554,23 +1554,23 @@ WHERE string_col IN ('null', 'array_null')"""))
         assertThat(query("""
                 SELECT COUNT(*)
                 FROM alltypes
-                WHERE long_col IS NULL"""))
-                .matches("VALUES (BIGINT '0')")
-                .isNotFullyPushedDown(FilterNode.class);
+                WHERE long_col IS NULL OR long_col = 0"""))
```

Comment:
Same here, should this be tested: if null handling is disabled, shouldn't `IS NULL` be always false, and if enabled it would be min value?